Streetcode/Streetcode.BLL/MediatR/Streetcode/Streetcode/GetPageMainPage/GetPageOfStreetcodesMainPageHandler.cs

```diff
@@ -0,0 +1,57 @@
+﻿using AutoMapper;
+using FluentResults;
+using MediatR;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Localization;
+using Streetcode.BLL.DTO.Streetcode;
+using Streetcode.BLL.Interfaces.Logging;
+using Streetcode.BLL.SharedResource;
+using Streetcode.BLL.Util;
+using Streetcode.DAL.Enums;
+using Streetcode.DAL.Repositories.Interfaces.Base;
+
+namespace Streetcode.BLL.MediatR.Streetcode.Streetcode.GetPageMainPage
+{
+    public class GetPageOfStreetcodesMainPageHandler : IRequestHandler<GetPageOfStreetcodesMainPageQuery,
+        Result<IEnumerable<StreetcodeMainPageDTO>>>
+    {
+        private readonly IMapper _mapper;
+        private readonly IRepositoryWrapper _repositoryWrapper;
+        private readonly ILoggerService _logger;
+        private readonly IStringLocalizer<NoSharedResource> _stringLocalizerNo;
+
+        public GetPageOfStreetcodesMainPageHandler(IRepositoryWrapper repositoryWrapper, IMapper mapper, ILoggerService logger, IStringLocalizer<NoSharedResource> stringLocalizerNo)
```

Comment:
please consider renaming to GetPageOfStreetcodes or something similar because backend should not me wired to any of frontend logic (Here in names is said that it's only for MainPage but it's not important at all here in backend controller)

Reply:
This request has MainPage in the name as far as it returns StreetcodeMainPageDTO entity. Isn't it an appropriate way to specify expected output?